git_latexdiff_branch.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
If called as `../git_latexdiff_branch.sh` or `thesis/git_latexdiff_branch.sh` nearly all commands will fail.

Since the script already expects to be alongside Makefile (which is reasonable), setting `repo_path` to the script's dir is safer:
```
repo_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
cd "$repo_path"
```

(The `cd` is necessary else git and make will still fail)

Alternatively, you could issue  `set -e` as the first command to make the script stop on first error, but then some side effects might already be visible.